tests/dummy/app/components/widgets/widget-3/tour-schedule/responsive-image.hbs

```diff
@@ -1,25 +1,14 @@
-{{#if @images}}
-  <ContainerQuery
-    @debounce={{300}}
-    as |CQ|
-  >
-    <div
-      local-class="image-container"
-      {{find-best-fitting-image
-        dimensions=CQ.dimensions
-        images=@images
-        onQuery=this.setImageSource
-      }}
-    >
-      {{#if this.imageSource}}
-        <img
-          alt=""
-          data-test-image="Concert"
-          local-class="image"
-          role="presentation"
-          src={{this.imageSource}}
-        />
-      {{/if}}
-    </div>
-  </ContainerQuery>
-{{/if}}
\ No newline at end of file
+<div
+  local-class="image-container"
+  {{container-query debounce=300 onQuery=this.setImageSource}}
```

Comment:
The component now provides an addition test for using `{{container-query}}` modifier as a standalone.